core/src/main/java/apoc/merge/Merge.java

```diff
@@ -33,10 +37,35 @@
     @Procedure(value="apoc.merge.node", mode = Mode.WRITE)
     @Description("\"apoc.merge.node.eager(['Label'], identProps:{key:value, ...}, onCreateProps:{key:value,...}, onMatchProps:{key:value,...}}) - merge nodes with dynamic labels, with support for setting properties ON CREATE or ON MATCH")
```

Comment:
so che non c'entra nulla ma per favore correggi la descrizione qui
```suggestion
    @Description("\"apoc.merge.node(['Label'], identProps:{key:value, ...}, onCreateProps:{key:value,...}, onMatchProps:{key:value,...}}) - merge nodes with dynamic labels, with support for setting properties ON CREATE or ON MATCH")
```